docs/deployment/integrations/kuberay.md

```diff
@@ -0,0 +1,27 @@
+---
+title: KubeRay
+---
+[](){ #deployment-kuberay }
+
+[KubeRay](https://github.com/ray-project/kuberay) provides a Kubernetes-native way to run vLLM workloads on Ray clusters.
+A Ray cluster can be declared in YAML, and the operator then handles pod scheduling, networking configuration, restarts, and blue-green deployments — all while preserving the familiar Kubernetes experience.
+
+---
```

Comment:
Just a style nit, but this would make this doc more consistent with the other docs

```suggestion
```